### PR TITLE
Priot test doccument

### PR DESCRIPTION
**Pull request by SvenStruanFinlay**
Code Cleanup and Documentation

### CODE REVIEW

**Comment by SvenStruanFinlay** [comment]
Dont need to check the files after the tests because its all the deleted files of the old solution. now its just cmake

File: Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/ml_interface.hpp [hunk 1]
```diff
@@ -14,15 +14,18 @@ struct ModelInterface {
 	float WHRatio = inWidth / (float)inHeight;
 	float inScaleFactor = 0.007843f;
 	float meanVal = 127.5;
-	float confidence_threshold = 0.8f;
+	float confidence_threshold = 0.7f;
```
Comment (GisliGeorgs) [inline Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/ml_interface.hpp]: Always lowering the threshold 😄

File: Intel_SighPP_CMake/Intel_SighPP_CMake/src/priority_lib/README.md [hunk 2]
```diff
@@ -1,28 +1,80 @@
...
 
+The system categorises the data into four priorities (Urgent, High, Medium and Low) based on aset of rule levels of the same name. The items are processed individually through levels of decision functions to determine a final priority. This can be seen in 8, focusing on objects size and distance. The Prioritiser has been designed with inheritance/interfacing to allow for modularity allowing future developers to create custom modules for other use cases. In our case the modules were designed to support the Sight++ PoC.Next, object tracking data was integrated into the prioritiser inclusive of speed analysis. It allowed the Prioritiser to understand new and known objects and manage the priority respectively. The prioritiser can now manage obstacle avoidance as well as detection. It does this by first prioritising for objects that may cause collisions and then, prioritising for significant environmental objects the user should be aware of.
```
Comment (GisliGeorgs) [inline Intel_SighPP_CMake/Intel_SighPP_CMake/src/priority_lib/README.md]: "can be seen in 8" doesn't point to anything, and below "in fig: 9 & 10" (the figures don't have numbers 😄 )

File: Intel_SighPP_CMake/Intel_SighPP_CMake/tests/test.cpp [hunk 3]
```diff
@@ -11,125 +11,199 @@
...
+#include "../src/priority_lib/smart_priority.hpp" 
+#include "../src/priority_lib/smart_priority.cpp"  
+#include "../src/priority_lib/priority_clock.cpp"   
+#include "../src/priority_lib/priority_clock.hpp"   
+#include "../src/priority_lib/priority_module.cpp"
+#include "../src/priority_lib/priority_module.hpp"
+#include "../src/classification_result.hpp"
+
+namespace {
+	class prioTest : public ::testing::Test {
```
Comment (GisliGeorgs) [inline Intel_SighPP_CMake/Intel_SighPP_CMake/tests/test.cpp]: These should probably be kept in separate files. Like test_prioritiser.cpp or something

File: README.md [hunk 4]
```diff
@@ -83,7 +104,23 @@ Flags for running the system include:
...
+
+The next step for the interface would be using a more adaptable STT API to satisfy various browsers. Although using speech APIs with buttons clicks can interact with users efficiently, it would be more convenient for users to use an application without buttons. Therefore, the next step could be to a more adaptable speech recognition APIs which could identify what the user speaks continuously and make corresponding interactions.
+
 ## Contrubuting
+All future Contributions should be submitted to the Intel Copy of this repo hosted by the Intel Company. The link to said repo will be posted here when it is up.
```
Comment (GisliGeorgs) [inline README.md:131]: When that happens this repo should probably be archived. With a link/notice to the other repo.

**Review by GisliGeorgs — CHANGES_REQUESTED** [review]
Please move prioritiser tests to separate file. Tests for different modules and components should probably/ideally kept in relevant files, not just jam everything into tests.cpp.

File: Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/ml_interface.hpp (hunk 1 above)
Comment (SvenStruanFinlay) [reply Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/ml_interface.hpp]: is good for demo!

File: Intel_SighPP_CMake/Intel_SighPP_CMake/tests/test.cpp (hunk 3 above)
Comment (SvenStruanFinlay) [reply Intel_SighPP_CMake/Intel_SighPP_CMake/tests/test.cpp]: done

File: Intel_SighPP_CMake/Intel_SighPP_CMake/src/priority_lib/README.md (hunk 2 above)
Comment (SvenStruanFinlay) [reply Intel_SighPP_CMake/Intel_SighPP_CMake/src/priority_lib/README.md]: done

File: README.md (hunk 4 above)
Comment (SvenStruanFinlay) [reply README.md:131]: ye, the whole thing is being transefered to the NHS open source library so i guess that's that lol

**Review by GisliGeorgs — APPROVED** [review]
Nice!

**Pull request merged**